test/cpp/thread_manager/thread_manager_test.cc

```diff
@@ -91,7 +91,7 @@ grpc::ThreadManager::WorkStatus ThreadManagerTest::PollForWork(void **tag,
   // Simulate "polling for work" by sleeping for sometime
   SleepForMs(kPollingTimeoutMsec);
 
-  *tag = nullptr;
+  *tag = NULL;
```

Comment:
Can you list the error that you got here? I think it's really inadvisable to use NULL in lieu of nullptr since NULL technically isn't a pointer and this is likely to choke some other compiler version.